### PR TITLE
Recommender frontend

### PR DESCRIPTION
**Pull request by AlephEleven**
added outfits/generate route:

allows you to choose clothes preferences (defined in data/gen_clothes.js), then saves & displays generated outfit, as a list of clothing cards.

### CODE REVIEW

**Review by LDomadia — CHANGES_REQUESTED** [review]
Which inputs are supposed to be required? Is it just the name or are all inputs required?
Screen Shot 2022-05-01 at 11 48 22 AM

also need to add a link from the My Outfits page to the Generate Outfit form page

**Comment by AlephEleven** [comment]
> Which inputs are supposed to be required? Is it just the name or are all inputs required? Screen Shot 2022-05-01 at 11 48 22 AM
> 
> also need to add a link from the My Outfits page to the Generate Outfit form page

got it, all parts are required, but there isnt a my outfits page yet so that can be done by whoevers making it

**Comment by AlephEleven** [comment]
updated also added outfits route

**Comment by AlephEleven** [comment]
fixed merge conflict

File: routes/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const outfitsRoutes = require("./outfits");
 const statsRoutes = require("./stats");
 const accountRoutes = require("./account");
 const profileRoutes = require("./profile");
+const outfitsRoutes = require("./outfits");
```
Comment (uityh) [inline routes/index.js]: there are two of these which cause an error

**Review by uityh — DISMISSED** [review]

**Review by LDomadia — APPROVED** [review]

**Pull request merged**